src/test/java/hometask3/VectorTest.java

```diff
@@ -0,0 +1,28 @@
+package hometask3;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class VectorTest {
```

Comment:
Please reformat this class (extra space before comas and '{') and add an error message to the assert in getScalarMultiplication() method
<img width="1652" alt="image" src="https://user-images.githubusercontent.com/90613721/180821306-e1057d41-9b17-44fe-a242-9875f39edc16.png">